### PR TITLE
Update `jellyfish-types` to v2 and fix type imports

### PR DESCRIPTION
**Pull request by Ereski**
Change-type: patch
Signed-off-by: Carol Schulze <carol@balena.io>

### CODE REVIEW

File: lib/card.spec.ts [hunk 1]
```diff
@@ -692,8 +696,8 @@ test('unlink will unlink all links between two cards with the specified verb', a
 	const cardSdk = new CardSdk(sdk as any);
 
 	await cardSdk.unlink(
-		account1 as any as core.Contract,
-		opportunity1 as any as core.Contract,
+		account1 as any as Contract,
+		opportunity1 as any as Contract,
```
Comment (karaxuna) [inline lib/card.spec.ts:700]: We should probably fix `unlink` method to accept something like this: `Pick<Contract, 'id' | 'type'>`, but not sure if it's under the scope of this pr.

Comment (Ereski) [reply lib/card.spec.ts:700]: Ya this `as any as Contract` is not great. I'll take a closer look :+1:

Comment (Ereski) [reply lib/card.spec.ts]: resolved by implementing your suggestion

**Comment by Ereski** [comment]
I'm going to assume it's ok to merge this

**Comment by Ereski** [comment]
@balena-ci I self-certify!

**Comment by Ereski** [comment]
@balena-ci retest

**Pull request merged**